crates/cubecl-runtime/src/server.rs

```diff
@@ -143,3 +142,29 @@ impl<Server: ComputeServer> Clone for Binding<Server> {
         }
     }
 }
+
+/// Provides launch information specifying the number of work groups to be used by a compute shader.
+pub enum CubeCount {
+    /// Dispatch x,y,z work groups.
+    Static(u32, u32, u32),
+    /// Dispatch work groups based on the values in this buffer. The buffer should contain a u32 array [x, y, z].
```

Comment:
I would avoid using work group here, since it's a different name for cuda. We could provide a list of equivalent terms in different languages (grid for cuda, num_workgroup for wgsl, etc.).

Reply:
Ah this was just copied, but yeah fixed up the docs a bit. This is user facing so used the 'cube' terminology and explained it a bit.